make.ps1

```diff
@@ -62,15 +59,16 @@ function create-smoke-tests() {
     . ./envrc.ps1
     $env:PACT_VERSION="$env:USERNAME".replace(' ','')
     $env:UPDATE_PRESCRIPTIONS=$update
-    $env:SIGNING_PRIVATE_KEY_PATH="../../../sign/eps_int_test_private.key"
-    $env:SIGNING_CERT_PATH="../../../sign/eps_int_test_certificate.crt"
+    $env:PACT_USE_BROKER=$broker
```

Comment:
Were we not going to remove the broker entirely for now? (We could probably do this as a new PR)

Reply:
I think we're dependant on it at the moment for both debugging failing smoke tests since the IDs and signatures are generated on each run and not persisted anywhere else and it's currently fulfilling the function of test reports. So I think we'll need it for a little longer till we have other solutions in place to address both of those points